go.mod

```diff
@@ -1,26 +1,13 @@
 module github.com/juju/aclstore/v2
 
+go 1.16
+
 require (
-	github.com/frankban/quicktest v0.8.0
-	github.com/google/go-cmp v0.1.0 // indirect
+	github.com/frankban/quicktest v1.2.2
```

Comment:
if you're updating this, might as well go for the newest version

Reply:
done